etl/steps/data/garden/shift/2022-07-18/fossil_fuel_production.py

```diff
@@ -12,18 +11,13 @@
 from owid.datautils import geo
 from structlog import get_logger
 
-from etl.paths import DATA_DIR
+from etl.helpers import Naming
 
 log = get_logger()
 
-NAMESPACE = "shift"
-DATASET_SHORT_NAME = "fossil_fuel_production"
 
-VERSION = Path(__file__).parent.name
-COUNTRY_MAPPING_PATH = (
-    Path(__file__).parent / f"{DATASET_SHORT_NAME}.country_mapping.json"
-)
-METADATA_PATH = Path(__file__).parent / f"{DATASET_SHORT_NAME}.meta.yml"
+N = Naming(__file__)
+
```

Comment:
Maybe it's better to call it something more descriptive, maybe `NAMES`?

Reply:
Good idea, renamed to `Names`.